### PR TITLE
Update: Better Watson credentials settings

### LINKED ISSUE #41: Better Watson credentials settings
Opening an issue to track this post-initial soft release:

- [x] Add inline help text about creating an account and a resource, e.g. under the header and before the fields:
> Don't have an IBM Cloud account yet? [Register for one](https://cloud.ibm.com/registration) and set up a [Natural Language Understanding](https://cloud.ibm.com/catalog/services/natural-language-understanding) Resource to get your API key.
- [x] Expand the length of the URL and API key fields (`large-text` class)
- [x] Hide the username field and just have `API URL` and `API Key` fields by default.
- [x] Add a link toggle underneath saying something like `I have a username/password instead` that then shows a `Username` field and changes `API Key` to `Password`

### PR DESCRIPTION
**Pull request by dinhtungdu**
### Description of the Change

* Add inline help text about creating an account and a resource, e.g. under the header and before the fields:
* Expand the length of the URL and API key fields (large-text class)
* Hide the username field and just have API URL and API Key fields by default.
* Add a link toggle underneath saying something like `I have a username/password instead` that then shows a Username field and changes API Key to Password

### Alternate Designs
n/a

### Benefits
Better Watson credentials settings.

### Possible Drawbacks
n/a

### Verification Process

1. Go to admin > ClassifAI > Language Processing.
2. See the new description under 'Natural Language Understanding'.
3. See credentials section now has only 2 fields: 'API URL' and 'API Key'.
4. See the new toggle link say: 'Use a username/password instead?'.
5. Click that toggle link, a new filed 'API Username' appears and has value that is `apikey`.
6. Change value of API Username to something different from `apikey` and save settings.
7. See the toggle link after page load is 'Use an API Key instead?'.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/10up/classifai/blob/develop/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [x] All new and existing tests passed.

### Applicable Issues
Resolves #41

### CODE REVIEW

File: includes/Classifai/Providers/Watson/NLU.php [hunk 1]
```diff
@@ -203,8 +227,26 @@ public function init_admin_scripts() {
...
 	public function setup_fields_sections() {
+		// Add the settings section.
+		add_settings_section(
+			$this->get_option_name(),
+			$this->provider_service_name,
+			function() {
+				printf(
+					wp_kses(
+						__( 'Don\'t have an IBM Cloud account yet? <a href="%1$s">Register for one</a> and set up a <a href="%2$s">Natural Language Understanding</a> Resource to get your API key.', 'classifai' ),
+						[ 'a' => [] ]
```
Comment (ryanwelcher) [inline includes/Classifai/Providers/Watson/NLU.php]: We're missing the `href` attribute here.

**Comment by ryanwelcher** [comment]
Great work on thins @dinhtungdu  - merge when ready!

**Comment by jeffpaul** [comment]
@dinhtungdu looks like GitHub had some networking issues as this got merged. Could you double check `develop` to ensure it's got your changes from this PR as expected?

**Comment by jeffpaul** [comment]
@dinhtungdu looks like GitHub had some networking issues as this got merged. Could you double check `develop` to ensure it's got your changes from this PR as expected?

**Pull request merged**